x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/slack/slack_connectors.tsx

```diff
@@ -47,7 +47,6 @@ const SlackActionFields: React.FunctionComponent<ActionConnectorFieldsProps<
             isInvalid={errors.webhookUrl.length > 0 && webhookUrl !== undefined}
             name="webhookUrl"
             readOnly={readOnly}
-            placeholder="Example: https://hooks.slack.com/services"
```

Comment:
nit: For a Slack webhook URL we should not be able to pass non https protocol or invalid URL. But we don't have URL validation on the API level and on the UI level. I think it would be nice to have it here. Anyway, we have a documentation for a Slack configuration.

Reply:
Added URL and protocol validation to the slack webhook url